Sources/Framework/Models/VoteContext.swift

```diff
@@ -16,6 +8,8 @@ public struct VoteContext {
 
 }
 
+// swiftlint:disable no_extension_access_modifier
```

Comment:
Please delete disabling this rule. Keyword public should be used here:

```swift
extension VoteContext {

      public func answer(for answerType: AnswerType) -> Answer {
...
```

instead of here:

```swift
public extension VoteContext {

      func answer(for answerType: AnswerType) -> Answer {
...
```